config.gradle

```
@@ -16,7 +17,7 @@ ext {
 
 // Parse the release version from the gradle project properties (e.g: -Prelease_version=<version>)
 ext.getReleaseVersion = { ->
-    final String defaultVersion = "0.1.0-dev-SNAPSHOT"
+    final String defaultVersion = "1.0.2-dev-SNAPSHOT"
```

The number moves forever upward!

```suggestion
    final String defaultVersion = "1.0.4-dev-SNAPSHOT"
```